src/browser.ts

```diff
@@ -0,0 +1,70 @@
+import { PaymentSdkConfigType } from "./types";
+
+import { XcooBeePaymentSDK } from "./XcooBeePaymentSDK";
+
+import { logoSrc } from "./config";
+
+/**
+ * Implements web version of Base64Interface to manipulate base64 strings
+ *
+ * @implements Base64Interface
+ */
+export const base64 = {
+  atob: (str: string): string => atob(str),
+  btoa: (str: string): string => btoa(str),
+};
+
+/**
+ * Implements web version of QrGeneratorInterface to generate QRs
+ *
+ * @implements QrGeneratorInterface
+ */
+export const qrGenerator = {
```

Comment:
Will the following work to help enforce it maintains the correct interface?
```suggestion
export const qrGenerator: QrGeneratorInterface = {
```